app/views/organisations/edit.html.erb

```diff
@@ -8,7 +8,7 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-l">
-        <%= content_for(:title) %>
+        <%= "Change #{@organisation.name}’s details" %>
```

Comment:
We would not want to use text returned by the sanitise_characters method to display on the page as the ampersand is slightly off (with extra padding). It's a compromise for the title but not needed on the page itself.

Reply:
That way we might avoid having to change the value of the title itself (?)